frontend_lib/src/component/Tags/NewTagForm.styl

```diff
@@ -0,0 +1,92 @@
+@import '../../css/Variable.styl'
+
+.taglist__form
+  padding 15px
+  flex-direction column
+  height 100%
+  width 100%
+  &__title
+    font-size 20px
+    margin-bottom 25px
+  &__tag
+    margin-bottom 10px
+  &__close
+    position absolute
+    right 20px
+    font-size 20px
+    z-index 1
+    & > i
+      cursor pointer
+  &__member
+    margin 0 0 20px 0
+    &__name
+      position relative
+      .name__label
+        margin 0 0 15px 0
+        font-weight bold
+      .name__input
+        margin 0 0 0 0
+      .name__adminmsg
+        margin 10px 0 10px 0
+        font-size 14px
+      .autocomplete
+        position absolute
+        width 300px
+        background-color offWhite
+        border-radius 10px
+        border-width 1px
+        border-style solid
+        z-index 1
+        &__item
+          display flex
+          align-items center
+          cursor pointer
+          padding 5px 8px
+          &:hover
+            background-color: grey-hover
+          &:first-child
+            border-top-left-radius 10px
+            border-top-right-radius 10px
+          &:last-child
+            border-bottom-left-radius 10px
+            border-bottom-right-radius 10px
+          &__avatar
+            width 45px
+            height 45px
+            border-radius 50%
+          &__icon
+            font-size 28px
+          &__name
+            margin-left 15px
+            overflow hidden
+            text-overflow ellipsis
+            white-space nowrap
+            &__username
+              font-size small
```

Comment:
Can you use the font size defined at Variable.styl here also?

Reply:
this is some dead code I'm removing.